examples/flutter_todos/android/app/src/main/kotlin/com/example/verygoodcore/MainActivity.kt

```diff
@@ -1,6 +1,6 @@
-package com.example.flutter_todos
+package com.example.verygoodcore
 
 import io.flutter.embedding.android.FlutterActivity
 
 class MainActivity: FlutterActivity() {
-}
+}
```

Comment:
my logaf is low on this one, but missing new line

Reply:
These files are generated by the Very Good CLI (and, in extension, `flutter create`), so we usually don't edit native files manually if we don't need to. Thank you for the suggestion however 😄